packages/enzyme-test-suite/test/shared/methods/wrapProp.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { expect } from 'chai';
+import wrap from 'mocha-wrap';
+
+import getAdapter from 'enzyme/build/getAdapter';
+
+export default function describeWrapProp({
+  Wrap,
+  WrapRendered,
+  WrapperName,
+  isShallow,
+}) {
+  wrap()
+    .withConsoleThrows()
+    .describe('.wrapProp()', () => {
+      class Inner extends React.Component {
+        render() {
+          return <div className="inner" />;
+        }
+      }
+      class Outer extends React.Component {
+        render() {
+          return <div />;
+        }
+      }
+      class Container extends React.Component {
+        render() {
+          const node = this.props.replace || <Inner />;
+          return <Outer node={node} />;
+        }
+      }
+
+      it('returns a wrapper around the node provided by the given prop', () => {
+        const wrapper = Wrap(<Container />);
+        const wrappedPropWrapper = wrapper.find(Outer).wrapProp('node');
+        expect(wrappedPropWrapper.find('div').equals(<div className="inner" />)).to.be.true;
```

Comment:
```suggestion
        expect(wrappedPropWrapper.find('div').equals(<div className="inner" />)).to.equal(true);
```
noop matchers should be avoided.